README.md

```diff
@@ -1,2 +1,322 @@
-# eos-etl
-ETL scripts for EOS, contributions are welcome
+# Bitcoin ETL
```

Comment:
Please go over this file and update where necessary. @Vasiliy-Bondarenko

Reply:
done